docs/modules/ROOT/pages/getting-started/filters-and-projections.adoc

```diff
@@ -0,0 +1,191 @@
+
+The second parameter is `"The Matrix"`, but instead of using a plain string, the object `Cypher.Param` class will take care of turning that into a parameter so we can safely put user input here.
+
+
+.Why `Cypher.eq` doesn't need `new`?
+[NOTE]
+====
+Cypher Builder treats Variables (like nodes and relationships) and Clauses as classes, as these are intended to be assigned to a JavaScript variable to be reused, composed and built.
+Cypher Functions and Procedures, however, are intended to be used a single time, as part of a larger query, so Cypher Builder treats these as simple functions, making it closer to how these work in Cypher.
+====
+
+=== Cypher and params
+
+You can run your script again (`node main.js`), and should get the following Cypher:
+
+```cypher
+MATCH (this0:Movie)
+WHERE this0.title = $param0
+RETURN this0
+```
+
+Success! The movie is now filtered by its title to a param. The param, however, is missing from the output. You'll need to tweak the build process at the end of the script to get the parameters as well:
+
+```javascript
+const { cypher, params } = match.build();
+console.log(cypher);
+console.log(params);
+```
+
+Now, executing again, we get all we need to run this query in Neo4j:
+
+```cypher
+MATCH (this0:Movie)
+WHERE this0.title = $param0
+RETURN this0
+```
+
+```javascript
+{ param0: 'The Matrix' }
+```
+
+[TIP]
+====
+Note that `new Param` already takes care of assigning a param name and correctly matching the value to that parameter in the Cypher query. If a `Param` instance is used multiple times within a query, it will always reference the same parameter (`$param0` in this case).
+====
+
+=== `AND` filtering
+
+The query still lacks the second filter, which needs to be concatenated with `AND`. Simply use the method `.and` after the `where` with a new `Cypher.eq` predicate:
+
+```javascript
+    .where(Cypher.eq(movieNode.property("title"), new Cypher.Param("The Matrix")))
+    .and(Cypher.lt(movieNode.property("released"), new Cypher.Param(2000)))
+```
+
+This takes the same parameters as `.where`, and makes sure to properly concatenate the filters with `AND`. In this case, the function `Cypher.lt` will add the operator `<` (**l**ower **t**han). Same as before, the comparison happens between a property of `movieNode` and a param. This time the param is a number. Cypher Builder will take care of correctly passing it to the `params` object.
+
+Running the script again:
+
+```cypher
+MATCH (this0:Movie)
+WHERE (this0.title = $param0 AND this0.released < $param1)
+RETURN this0
+```
+
+```javascript
+{ param0: 'The Matrix', param1: 1999 }
+```
+
+This should look closer to the target query, note that now there are two parameters, each one correctly associated to the correct filter.
+
+[NOTE]
+====
+Adding the `AND` operation automatically adds parenthesis to ensure operation precedence, this will be important when dealing with complex and nested filters 
+====
+
+== Projection
```

Comment:
I have the feeling this section should be mentioned in the introductory paragraph and maybe have some explanation of what projection means or is this a common term?

Reply:
It is briefly mentioned: `in this guide you'll add some filters and change the projection to only return some fields`

I assume people are roughly familiar with what projection means